ipmininet/cli.py

```diff
@@ -81,6 +81,18 @@ def do_ping6pair(self, _line):
         """Ping (IPv6-only) between first two hosts, useful for testing."""
         self.mn.ping6Pair()
 
+    def do_link(self,line):
```

Comment:
Please use type hinting (https://docs.python.org/3/library/typing.html) for your methods

For instance, here, I think that it is
```suggestion
    def do_link(self, line: str):
```